tika-parsers/tika-parsers-classic/tika-parsers-classic-modules/tika-parser-ocr-module/src/main/java/org/apache/tika/parser/ocr/tess4j/ImageDeskew.java

```diff
@@ -0,0 +1,135 @@
+package org.apache.tika.parser.ocr.tess4j;
```

Comment:
Please add Apache License header.

Also, please add a comment along the lines of "copied and pasted from" and then a link to the tess4j code